ament_cmake_export_libraries/cmake/environment/library_path.bat.in

```diff
@@ -31,4 +29,3 @@ goto:eof
     set "%~1=%_list%"
   )
 goto:eof
-
```

Comment:
Because the changes are across repos and there I can't combine them in a single PR.